tests/paras/paras.toml

```
@@ -33,6 +33,16 @@ chain = "bifrost-kusama-local"
   args = ["-lparachain=debug"]
 
 
+[[parachains]]
+id = 2002
+chain = "turing-dev"
+cumulus_based = true
+
+  [parachains.collator]
+  name = "alister"
+  image = "{{OAK_COL_IMAGE}}"
+  command = "oak-collator"
```

```suggestion
  command = "/oak/oak-collator"
```

The binary is not in the PATH (https://grafana.parity-mgmt.parity.io/goto/Srl18Db4z?orgId=1)